semantic/src/site/globals/site.variables

```diff
@@ -50,6 +50,9 @@
 @headersInteractiveMed: #656565;
 @borderColourDark: #777777;
 @activeCommentBackground: azure;
+@inputPlaceholderColor: lighten(@inputColor, 20);
+@inputPlaceholderFocusColor: lighten(@inputColor, 40);
+@disabledOpacity: 0.75;
```

Comment:
This makes the non-editable text less light, and the placeholder text too. Please let me know if you find any cases where this change seems inappropriate.